modules/gitops-namespace-scoped-instance.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * multitenancy/multitenancy-support-in-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-namespace-scoped-instance_{context}"]
+= Namespace-scoped instance (Application delivery instance)
+
+After you create an Argo CD custom resource (CR) in one of your namespaces, the {gitops-shortname} Operator brings up an Argo CD in this namespace which you can use for application delivery. In its initial state, this instance has permission to deploy resources only in the same namespace it is installed to. However, there are times when organizations have specific requirements to satisfy that require additional tuning.
```

Comment:
```suggestion
After you create an Argo CD custom resource (CR) in one of your namespaces, the {gitops-shortname} Operator starts an Argo CD in this namespace, which you can use for application delivery. In its initial state, this instance has permission to deploy resources only in the same namespace in which it is installed. However, there are times when organizations have specific requirements to satisfy that require additional tuning.
```
I was confused by the last sentence: does "organization" refer to the user who created the Argo CD CR, or to a tenant/additional party for whom the installer needs to tune the Argo CD instance? Also, does "tuning" just mean configuration, or something more specific? I'd suggest changing the last sentence to something like this: "However, you may need to configure the instance to meet your specific requirements."